app/renderer/components/ErrorBoundary/ErrorMessage.js

```diff
@@ -2,13 +2,12 @@ import {CopyOutlined} from '@ant-design/icons';
 import {Alert, Button, Tooltip} from 'antd';
 import React from 'react';
 
+import {ALERT} from '../../constants/ANTD_TYPES';
+import {LINKS} from '../../constants/COMMON';
```

Comment:
Not sure we need to have uppercase file names. Is it mentioned in any standard?

Reply:
I saw this as a suggestion when searching for best practices for constants, and I thought it made sense. But it's probably not specified in any standard, no.

Reply:
lets only use lowercase for file names. It is already clear these are constants as they are located in the `constants` subdir